src/api.ts

```diff
@@ -0,0 +1,20 @@
+import { TodoInterface, UserInterface, PreparedTodoInterface } from './interfaces';
+
+const API_USERS = 'https://mate.academy/students-api/users';
+const API_TODOS = 'https://mate.academy/students-api/todos';
+
+const fetchData = async (url: string) => {
+  const dataObject = await fetch(url).then(response => response.json());
+
+  return dataObject.data;
+};
+
+export const getData = async (): Promise<PreparedTodoInterface[]> => {
```

Comment:
```suggestion
export const getData = async (): Promise<]Todo[]> => {
```
Не нужно называть интерфейс интерфейсом, и Prepared не о чем не говорит, это todo которая включает в себя user, назови соответственно.

Reply:
исправь везде в соответствии с коментом выше